src/pages/what-is-ebpf.js

```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+	    <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+	    there was a vote taken and the bee was named **eBee**.
+	    For details acceptable uses of the logo, please see our
+	    <a href="https://ebpf.io/brand-guidelines/"> Brand Guidelines </a>
```

Comment:
Looks like the file is using spaces for indent.

```suggestion
            <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
            there was a vote taken and the bee was named **eBee**.
            (For details on acceptable uses of the logo, please see our
            <a href="https://ebpf.io/brand-guidelines/">Brand Guidelines</a>.)
```